static/elements/chromedash-approvals-dialog.js

```diff
@@ -405,6 +523,16 @@ class ChromedashApprovalsDialog extends LitElement {
   handleCancel() {
     this.shadowRoot.querySelector('chromedash-dialog').close();
   }
+
+  toggleConfig(approvalDef) {
+    let newConfigs = new Set([...this.showConfigs]); // Make a copy.
```

Comment:
Why are we making a copy? Is it to trigger certain functions?

Reply:
Yes, it is to trigger a new call to render(). 

My understanding is that modifying a collection is not detected as a change to a lit-element property.  Instead, we need to set the property itself to point to a new collection.  This page is not explicit about it, but it says that the first step in the process is that `the property setter is called`.  That is not going to happen if we simply mutate an existing property value.
https://lit.dev/docs/v1/components/properties/#what-happens-when-properties-change

Reply:
`That is not going to happen if we simply mutate an existing property value.` 
> Yup that is the case for Polymer as well. Thanks for the explanation